dipy/workflows/segment.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import division, print_function, absolute_import
+
+from glob import glob
+from os.path import join, basename, splitext, dirname, isabs, exists
+from os import makedirs
+
+import nibabel as nib
+import numpy as np
+
+from dipy.segment.mask import median_otsu
+
+
+def median_otsu_bet(input_file, out_dir, save_masked=False, median_radius=4,
```

Comment:
Of course you will need to write a docstring with an explanation of the parameters and perhaps an example of usage. The good news is that most of this information can then be copied in the dipy/bin file.